tests/drivers/pwm/pwm_api/boards/milkv_duo.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023-2024 Chen Xingyu <hi@xingrz.me>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		pwm-0 = &pwm1;
+	};
+};
+
+&pinctrl {
+	pwm1_default: pwm1_default {
+		group1 {
+			pinmux = <CVI_PINMUX(SD1_D2, PWM_5)>;  /* GP4 */
+		};
+	};
+};
+
+&pwm1 {
+	status = "okay";
+	pinctrl-0 = <&pwm1_default>;
+	pinctrl-names = "default";
+};
```

Comment:
any reason to not define this at board level?

Reply:
Well, I'm not entirely sure about when it should be defined at the board level versus when it should be an overlay for a test. If I understand correctly, all nodes should be 'disabled' by default and only enabled via overlay when needed, right? So, if all pins are GPIO by default, then PWMs should only be enabled where they are actually used?

Reply:
is this pin meant to always be used for PWM in this board? or is it a random pin chosen for testing only?

Reply:
It's a random pin I choose for testing only.